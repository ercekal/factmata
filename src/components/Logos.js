import React from 'react';
import styled from 'styled-components';
import Email from '../images/Email'
import Globe from '../images/Globe'

const Div = styled.div`
  display: flex;
  width: 60px;
  justify-content: space-between;
`

const Logos = ({website, email}) => {
  return (
    <Div>
      <a href={`http://www.${website}`} target='_blank' rel="noopener noreferrer">
        <Globe />
      </a>
      <a href={`mailto:${email}`}>
        <Email />
      </a>
    </Div>
  );
};

export default Logos;