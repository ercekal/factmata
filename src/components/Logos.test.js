import React from 'react';
import Logos from './Logos';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Logos email="test@email.com" website='test.com' />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
