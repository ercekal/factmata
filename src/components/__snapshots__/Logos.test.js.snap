// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa dlBMjn"
>
  <a
    href="http://www.test.com"
    rel="noopener noreferrer"
    target="_blank"
  >
    <svg
      clipRule="evenodd"
      fillRule="evenodd"
      height="24"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M12 0c6.623 0 12 5.377 12 12s-5.377 12-12 12-12-5.377-12-12 5.377-12 12-12m2.557 16h-5.115c.546 2.46 1.441 4.114 2.558 5.744 1.194-1.741 2.041-3.41 2.557-5.744m-7.157 0h-4.567c1.236 2.825 3.704 4.972 6.755 5.716-1.048-1.733-1.783-3.658-2.188-5.716m13.767 0h-4.567c-.391 1.988-1.095 3.887-2.175 5.694 3.012-.763 5.517-2.895 6.742-5.694m-14.005-6h-4.962c-.267 1.313-.267 2.685 0 4h4.915c-.119-1.329-.101-2.672.047-4m7.661 0h-5.647c-.165 1.326-.185 2.672-.053 4h5.753c.133-1.328.111-2.673-.053-4m6.977 0h-4.963c.148 1.328.166 2.671.048 4h4.915c.26-1.285.273-2.648 0-4m-12.156-7.729c-3.077.732-5.567 2.886-6.811 5.729h4.653c.435-2.042 1.178-3.985 2.158-5.729m2.355-.048c-1.089 1.77-1.91 3.453-2.463 5.777h4.927c-.534-2.246-1.337-3.948-2.464-5.777m2.368.069c1.013 1.812 1.733 3.76 2.146 5.708h4.654c-1.232-2.816-3.762-4.958-6.8-5.708"
        fill="hsl(204, 86%, 53%)"
      />
    </svg>
  </a>
  <a
    href="mailto:test@email.com"
  >
    <svg
      height={24}
      viewBox="0 0 24 24"
      width={24}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 3v18h24v-18h-24zm6.623 7.929l-4.623 5.712v-9.458l4.623 3.746zm-4.141-5.929h19.035l-9.517 7.713-9.518-7.713zm5.694 7.188l3.824 3.099 3.83-3.104 5.612 6.817h-18.779l5.513-6.812zm9.208-1.264l4.616-3.741v9.348l-4.616-5.607z"
        fill="hsl(204, 86%, 53%)"
      />
    </svg>
  </a>
</div>
`;
